### PR TITLE
Rename ExpensiPicker

### PR DESCRIPTION
**Pull request by mateusbra**
### Details

Rename component `ExpensiPicker` to `Picker`, rename folder [Picker](https://github.com/Expensify/App/tree/main/src/components/Picker) to NativePicker in order to solve import conflicts
### Fixed Issues

$ https://github.com/Expensify/App/issues/6760

### Tests

1. Run the app and see that Picker still working
### QA Steps

1. Run the app and see that Picker still working
### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

Web

#### Mobile Web

Mobile Web

#### Desktop

Desktop

#### iOS

IOS

#### Android

Android

### CODE REVIEW

File: src/components/NativePicker/pickerStyles/index.android.js
Comment (parasharrajat) [inline src/components/NativePicker/pickerStyles/index.android.js:4]: I understand that `ctrl-shift+F` works great. But please change it to camelCase and other instances if any.

File: src/components/Picker.js
Comment (parasharrajat) [inline src/components/Picker.js:5]: Ok. Let's do one thing. 
2. Rename it to BasePicker and move the whole NativePicker folder to Picker folder.
3. Move Picker.js to Picker/index.js
4. Create Picker Folder.

**Review by parasharrajat — CHANGES_REQUESTED** [review]

File: src/components/Picker/BasePicker/index.js
Comment (parasharrajat) [inline src/components/Picker/BasePicker/index.js:8]: BasePicker

File: src/components/Picker/BasePicker/index.js
Comment (parasharrajat) [inline src/components/Picker/BasePicker/index.js:31]: BasePicker

File: src/components/Picker/BasePicker/index.js
Comment (parasharrajat) [inline src/components/Picker/BasePicker/index.js:33]: same

File: src/components/Picker/BasePicker/index.js
Comment (parasharrajat) [inline src/components/Picker/BasePicker/index.js:6]: Let's prefix these files with `base` if they are only used for basePicker.

**Review by parasharrajat — CHANGES_REQUESTED** [review]

**Comment by mateusbra** [comment]
@parasharrajat should we do the same to all the index.js inside the pickerstyles folder?
[index.js](https://github.com/mateusbra/Expensify.cash/blob/rename-ExpensiPicker/src/components/Picker/BasePicker/pickerStyles/index.js),
[index.ios.js](https://github.com/mateusbra/Expensify.cash/blob/rename-ExpensiPicker/src/components/Picker/BasePicker/pickerStyles/index.ios.js),
[index.android.js](https://github.com/mateusbra/Expensify.cash/blob/rename-ExpensiPicker/src/components/Picker/BasePicker/pickerStyles/index.android.js)

**Comment by parasharrajat** [comment]
Doesn't matter much. I leave this up to you.

**Comment by mateusbra** [comment]
Ok, so I think thats it, just need your approval 😄

**Review by parasharrajat — DISMISSED** [review]
LGTM 

cc; @puneetlath

**Review by puneetlath — DISMISSED** [review]
@mateusbra just like the other PR. Thanks for your patience through the holidays. Everything looks good to me, but you've picked up some merge conflicts. Once you fix those I can quickly re-approve and merge. Thanks!

**Comment by mateusbra** [comment]
@puneetlath I think we got a problem with ExpensifyText, it looks like we still have a file calling ExpensifyText instead of Text

**Comment by mateusbra** [comment]
you think its ok to deal about it in this PR?

**Comment by mateusbra** [comment]
> @puneetlath I think we got a problem with ExpensifyText, it looks like we still have a file calling ExpensifyText instead of Text

looks like some PR was merged using ExpensifyText instead of Text after we merged the ExpensifyText PR

**Comment by parasharrajat** [comment]
@mateusbra Could you fix that issue as you are in charge of these changes? I don't think we mind one extra change in this PR if it is all about renaming.

**Comment by mateusbra** [comment]
thanks @parasharrajat 😄

File: android/app/build.gradle
Comment (parasharrajat) [inline android/app/build.gradle:153]: How are these changes related to this PR?

**Review by parasharrajat — CHANGES_REQUESTED** [review]

File: android/app/build.gradle
Comment (parasharrajat) [reply android/app/build.gradle:153]: I am not sure. But when we merge main, do the new changes show in PR changes?

Comment (mateusbra) [reply android/app/build.gradle:153]: I think they shoudn't show, I'm not understanding why its showing, we are currently using version 1.1.24-22 it shouldn't show as a change

Comment (mateusbra) [reply android/app/build.gradle:153]: I'll try to revert the last commit changes

**Review by parasharrajat — APPROVED** [review]
Seems like the test will pass. SO LGTM. Nice work. 

cc: @puneetlath 
:ribbon: :eyes: :ribbon:  C+ reviewed

**Review by puneetlath — CHANGES_REQUESTED** [review]
@mateusbra when I try to compile it it's failing because it seems there is still a reference to `ExpensifyText` in `src/pages/home/report/ReportActionItemMessage.js` even though there is no longer an `ExpensifyText` component.

**Comment by puneetlath** [comment]
Actually, looks like that is addressed here: https://expensify.slack.com/archives/C01GTK53T8Q/p1641299616468900

**Review by puneetlath — APPROVED** [review]

**Pull request merged**
